middleware/logger.go

```diff
@@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
 				code = color.Cyan(n)
 			}
 
-			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
+			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
```

Comment:
I am outside, you guys can discuss send me the transcript. 

Sent from my iPhone

> On Sep 18, 2015, at 7:54 AM, axdg <notifications@github.com> wrote:
> 
> In middleware/logger.go:
> 
> > @@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
> >  				code = color.Cyan(n)
> >  			}
> >  
> > -			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
> > +			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
> Header is fine, it obviously useful if your running behind a proxy. But as it stands, the values that are being logged would appear to have been chosen arbitrarily. At some point it’ll need a refactor, as vishr said. P.S. do you guys have some time to discuss #205 in glitter? On 19 September 2015 at 12:50:50 am, husobee (notifications@github.com) wrote: In middleware/logger.go:
> @@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc { code = color.Cyan(n) } - log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size) + log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
> will do, thanks vishr. — Reply to this email directly or view it on GitHub.
> —
> Reply to this email directly or view it on GitHub.
>